Double registration causing CAPA controller manager to panic

/kind bug

**What steps did you take and what happened:**
1. `make generate-examples`
1. `clusterctl create cluster -v 4 --bootstrap-type kind -c examples/_out/cluster.yaml -m examples/_out/controlplane.yaml -p examples/_out/provider-components.yaml -a examples/addons.yaml`
1. Once the bootstrap cluster is up and running, you will notice that the capa-controller-manager is crash looping.

**What did you expect to happen:**
1. To **not** have the capa-controller-manager CrashLoopBackoff

**Anything else you would like to add:**

```
$ kubectl logs -l control-plane=capa-controller-manager -n capa-system --tail 100 -f
panic: Double registration of different types for infrastructure.cluster.x-k8s.io/v1alpha2, Kind=AWSCluster: old=sigs.k8s.io/cluster-api-provider-aws/api/v1alpha2.AWSCluster, new=sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3.AWSCluster in scheme "pkg/runtime/scheme.go:101"

goroutine 1 [running]:
k8s.io/apimachinery/pkg/runtime.(*Scheme).AddKnownTypeWithName(0xc0003e8700, 0x1a67902, 0x1f, 0x1a47aef, 0x8, 0x16379f0, 0xa, 0x1d50000, 0xc0002ba380)
        /go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/runtime/scheme.go:203 +0xa03
k8s.io/apimachinery/pkg/runtime.(*Scheme).AddKnownTypes(0xc0003e8700, 0x1a67902, 0x1f, 0x1a47aef, 0x8, 0xc0002943a0, 0x2, 0x2)
        /go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/runtime/scheme.go:180 +0x258
sigs.k8s.io/controller-runtime/pkg/scheme.(*Builder).Register.func1(0xc0003e8700, 0x0, 0x0)
        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/scheme/scheme.go:72 +0x76
k8s.io/apimachinery/pkg/runtime.(*SchemeBuilder).AddToScheme(0x2d14380, 0xc0003e8700, 0x0, 0x0)
        /go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190817020851-f2f3a405f61d/pkg/runtime/scheme_builder.go:29 +0x6d
sigs.k8s.io/controller-runtime/pkg/scheme.(*Builder).AddToScheme(...)
        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.3.0/pkg/scheme/scheme.go:87
main.init.0()
        /workspace/main.go:49 +0x62

```
This is the image being used:
```
    Image:         gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller:latest
    Image ID:      gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller@sha256:f069f4ef82c60f40900ba8e35cfdf38cbc522bb0ed414b54262c5b9bbc0442a3
```

--- a/api/v1alpha3/groupversion_info.go
+++ b/api/v1alpha3/groupversion_info.go
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "infrastructure.cluster.x-k8s.io", Version: "v1alpha2"}
+	GroupVersion = schema.GroupVersion{Group: "infrastructure.cluster.x-k8s.io", Version: "v1alpha3"}
 
 	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
 	SchemeBuilder = &scheme.Builder{GroupVersion: GroupVersion}

